base/shared/src/main/scala/scalaz/data/KleisliSyntax.scala

```diff
@@ -0,0 +1,46 @@
+package scalaz
+package data
+
+import Prelude._
+
+trait KleisliSyntax {
+  import Kleisli.{ runKleisli, wrapKleisli }
+
+  implicit class ToKleisliOps[F[_], A, B](k: Kleisli[F, A, B]) {
+
+    def hoist[G[_]](η: F ~> G): Kleisli[G, A, B] =
+      Kleisli.hoist(k)(η)
+
+    def andThen[C](j: Kleisli[F, B, C])(implicit B: Bind[F]): Kleisli[F, A, C] =
+      Kleisli.compose(j, k)
+
+    def compose[E](j: Kleisli[F, E, A])(implicit B: Bind[F]): Kleisli[F, E, B] =
+      Kleisli.compose(k, j)
+
+    def >=>[C](j: Kleisli[F, B, C])(implicit B: Bind[F]): Kleisli[F, A, C] =
+      Kleisli.compose(j, k)
+
+    def <=<[E](j: Kleisli[F, E, A])(implicit B: Bind[F]): Kleisli[F, E, B] =
+      Kleisli.compose(k, j)
+
+    def >>>[C](j: Kleisli[F, B, C])(implicit B: Bind[F]): Kleisli[F, A, C] =
+      Kleisli.compose(j, k)
+
+    def =<<(fa: F[A])(implicit B: Bind[F]): F[B] =
+      B.flatMap(fa)(runKleisli(k))
+
+    def ***[C, D](j: Kleisli[F, C, D])(
+      implicit S: Strong[Kleisli[F, ?, ?]],
+      B: Bind[F]
+    ): Kleisli[F, (A, C), (B, D)] =
+      S.first(k) >>> S.second(j)
+
+    def &&&[C](j: Kleisli[F, A, C])(
+      implicit S: Strong[Kleisli[F, ?, ?]],
+      M: Monad[F]
+    ): Kleisli[F, A, (B, C)] =
```

Comment:
Neither is `Strong` needed here, and `Monad` can be relaxed to `Bind`.

Reply:
I'll relax these and implement `first` and `second` by hand

Reply:
Nice!

But then `Apply` must be sufficient for `***` as well.

Reply:
Yes it is 😄 

Next order of business is definitely filling out `Apply` - I notice we only have `ap` available